cmd/proxy/Dockerfile

```diff
@@ -16,7 +16,8 @@ ARG VERSION="unset"
 
 RUN DATE="$(date -u +%Y-%m-%d-%H:%M:%S-%Z)" && GO111MODULE=on CGO_ENABLED=0 GOPROXY="https://proxy.golang.org" go build -ldflags "-X github.com/gomods/athens/pkg/build.version=$VERSION -X github.com/gomods/athens/pkg/build.buildDate=$DATE" -o /bin/athens-proxy ./cmd/proxy
 
-FROM alpine
+ARG ALPINE_VERSION=3.9.5
```

Comment:
might as well upgrade it to 3.11?

Reply:
@marwan-at-work done, I upgraded to 3.11.5